From 0fd652c6a70882fb9afeedc12af34d22147de573 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:39:49 -0400
Subject: [ethernet] i40e/i40evf: Bump i40e to 0.4.19 and i40evf to 0.9.38

Message-id: <1409658042-22689-229-git-send-email-sassmann@redhat.com>
Patchwork-id: 90429
O-Subject: [RHEL7.1 PATCH 228/281] i40e/i40evf: Bump i40e to 0.4.19 and i40evf to 0.9.38
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Catherine Sullivan <catherine.sullivan@intel.com>

Bump versions.

Change-ID: Id5082d7c3995fbddd22b3e303d804c86fcd240a3
Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 67b807e834463ca295bc79c94af4e8fb6db55ee6)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_main.c b/drivers/net/ethernet/intel/i40e/i40e_main.c
index e6549be..09b73e6 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -39,7 +39,7 @@ static const char i40e_driver_string[] =
 
 #define DRV_VERSION_MAJOR 0
 #define DRV_VERSION_MINOR 4
-#define DRV_VERSION_BUILD 17
+#define DRV_VERSION_BUILD 19
 #define DRV_VERSION __stringify(DRV_VERSION_MAJOR) "." \
       __stringify(DRV_VERSION_MINOR) "." \
       __stringify(DRV_VERSION_BUILD)    DRV_KERN
-- 
1.7.1